client/src/tutorials/TodoAppSeahorse/files/index.test.ts.raw

```diff
@@ -0,0 +1,68 @@
+
+describe("Todo App", async () => {
+  const task: string = "ship product";
+  const last_todo: anchor.BN = new anchor.BN(0);
```

Comment:
Same for types, let's also use camelCase in TypeScript (other references also need to be updated).

```suggestion
  const lastTodo = new anchor.BN(0);
```